Engine/Src/Ancona/Framework/Screens/AbstractScreen.cpp

```diff
@@ -58,3 +58,8 @@ void AbstractScreen::Exiting(float delta)
     _screenManager.Window.setView(_defaultCam);
     _screenManager.Window.draw(_transitionRect);
 }
+
+void AbstractScreen::SetTransitionRect(int width, int height)
```

Comment:
Instead of creating this, we should use the screenManager reference in the constructor to do the right thing at construction time.